Application/__tests__/BackToLanding.test.tsx

```diff
@@ -3,7 +3,14 @@ import { Logo } from '@/components/Logo';
 import { MantineProvider } from '@mantine/core';
 
 jest.mock('next/router', () => ({
-  useRouter: jest.fn(),
+  useRouter: jest.fn().mockReturnValue({
+    push: jest.fn(),
+  }),
+}));
+
+// Mocking @clerk/nextjs
+jest.mock('@clerk/nextjs', () => ({
+  useUser: jest.fn(() => ({ user: null })),
```

Comment:
Back-integrating the clerk authentication - this should be expected since this test is created pre-authentication age.